libs/langchain-mongodb/tests/integration_tests/test_retrievers.py

```diff
@@ -130,7 +116,7 @@ def test_hybrid_retriever(indexed_vectorstore: PatchedMongoDBAtlasVectorSearch)
         top_k=3,
     )
 
-    query1 = "What was the latest city that I visited?"
+    query1 = "What did I visit France?"
```

Comment:
This is interesting. Why did you change it? It wasn't working with the local model?

Reply:
Correct, I had to choose something easier to infer.